Formula/opencoarrays.rb

```diff
@@ -51,5 +51,7 @@ def install
     EOS
     system "#{bin}/caf", "tally.f90", "-o", "tally"
     system "#{bin}/cafrun", "-np", "3", "--oversubscribe", "./tally"
+    %r{/open-mpi/(?<openmpi_version>[\d._]*)/lib} =~ shell_output("#{bin}/caf --show")
+    assert_equal openmpi_version, Formula["open-mpi"].pkg_version.to_s
```

Comment:
How about this?
```suggestion
    assert_match Formula["open-mpi"].version.to_s, shell_output("#{bin}/caf --show")
```

Reply:
I think it should be `pkg_version` to compare `version` and `revision` at the same time.

Reply:
Maybe this is better then
```suggestion
    assert_match Formula["open-mpi"].prefix, shell_output("#{bin}/caf --show")
```

Reply:
`Formula["open-mpi"].prefix` would return `/home/linuxbrew/.linuxbrew/opt/open-mpi`

`caf --show` uses `/home/linuxbrew/.linuxbrew/Cellar/open-mpi/4.1.4_1/lib`

Reply:
Unless we can call `Formula["open-mpi"].versioned_prefix`, which is a private method.

Reply:
it doesn't for me, that would be `Formula["open-mpi"].opt_prefix`
```console
irb(main):001:0> Formula["open-mpi"].prefix
=> #<Pathname:/usr/local/Cellar/open-mpi/4.1.4_2>
```

We can also use `Formula["open-mpi"].lib`

Reply:
## ghcr.io/homebrew/ubuntu22.04:master

```
irb(main):001:0> Formula["open-mpi"].prefix
=> #<Pathname:/home/linuxbrew/.linuxbrew/opt/open-mpi>
irb(main):002:0> Formula["open-mpi"].lib
=> #<Pathname:/home/linuxbrew/.linuxbrew/opt/open-mpi/lib>
```
## brew config
```
HOMEBREW_VERSION: 3.6.1-33-g65da808
ORIGIN: https://github.com/Homebrew/brew
HEAD: 65da808a104d1cab0a6d1c5dde46b6ce85d60696
Last commit: 22 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d0bc876efc548fd780a06b218b77ada39ad1c25c
Core tap last commit: 73 minutes ago
Core tap branch: opencoarrays
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DEVELOPER: set
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_AUTO_UPDATE: set
Homebrew Ruby: 2.6.8 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: octa-core 64-bit unknown_0x6_0x8c
Clang: N/A
Git: 2.37.3 => /bin/git
Curl: 7.81.0 => /bin/curl
Kernel: Linux 5.15.0-47-generic x86_64 GNU/Linux
OS: Unknown
Host glibc: 2.35
/usr/bin/gcc: 11.2.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
xorg: N/A
```

Reply:
@Homebrew/brew is this intentional?

Reply:
> @Homebrew/brew is this intentional?

Yes.

Do
```ruby
Formula["open-mpi"].prefix.realpath
```